apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -157,9 +160,41 @@ export default class ResearchOutputs implements ResearchOutputController {
       authors.map((author) => this.associateResearchOutputToAuthors(author)),
     );
 
+    const { queryResearchTagsContentsWithTotal } =
+      await this.squidexGraphqlClient.request<
+        FetchResearchTagsQuery,
+        FetchResearchTagsQueryVariables
+      >(FETCH_RESEARCH_TAGS, {
+        top: 20,
```

Comment:
why the hard limit is 20?
what if there will be 21 tags - which is possible for users and RO

Reply:
as per requirement atm there are only 23 tags - i have set the limit to a hard 100.